packages/devtools_app/lib/src/shared/service.dart

```diff
@@ -24,20 +24,21 @@ Future<VmServiceWrapper> _connectWithSse(
       ? uri.replace(scheme: 'http')
       : uri.replace(scheme: 'https');
   final client = SseClient('$uri');
-  final Stream<String> stream = client.stream?.asBroadcastStream();
+  final Stream<String> stream =
+      client.stream?.asBroadcastStream() as Stream<String>;
   final service = VmServiceWrapper.fromNewVmService(
     stream,
-    client.sink.add,
+    client.sink!.add,
     uri,
   );
 
-  unawaited(client.sink?.done?.whenComplete(() {
+  unawaited(client.sink?.done.whenComplete(() {
```

Comment:
if sink is nullable, then don't we still need the `?` after `done`? what if `client.sink` is null?

Reply:
we don't: https://dartpad.dev/?id=b0607a3f57081b35099f702c55bc82bd